packages/docsify-server-renderer/index.js

```diff
@@ -1,18 +1,34 @@
 import { readFileSync } from 'fs';
 import { resolve, basename } from 'path';
 import resolvePathname from 'resolve-pathname';
+import fetch from 'node-fetch';
+import debug from 'debug';
+import DOMPurify from 'dompurify';
 import { AbstractHistory } from '../../src/core/router/history/abstract';
 import { Compiler } from '../../src/core/render/compiler';
 import { isAbsolutePath } from '../../src/core/router/util';
 import * as tpl from '../../src/core/render/tpl';
 import { prerenderEmbed } from '../../src/core/render/embed';
-import fetch from 'node-fetch';
-import debug from 'debug';
 
 function cwd(...args) {
   return resolve(process.cwd(), ...args);
 }
 
+// Borrowed from https://j11y.io/snippets/getting-a-fully-qualified-url.
+function qualifyURL(url) {
+  const img = document.createElement('img');
```

Comment:
document will be undefined in server !

Reply:
Ah ok. Let me see....

Reply:
cc @trusktr document will be undefined

Reply:
We should be fine, now that I verified we have `JSDOM` running on the server-side. Let me verify if the `img` trick works. Also should write a test while we're at it.